From 95ac4e7be856343240147eda6d1479a61e0e020f Mon Sep 17 00:00:00 2001
From: Celeste Liu <CoelacanthusHex@gmail.com>
Date: Mon, 16 Sep 2024 00:31:41 +0800
Subject: [PATCH] hwdb: add Kensington SlimBlade Pro trackball (Bluetooth mode)

Wired and 2.4G dongle connectivity is covered by general trackball rule,
but with Bluetooth connectivity Kensington SlimBlade Pro uses the name
"SlimBlade Pro" which doesn't contain "[Tt]rack[Bb]all". We need to
process it specially.

Signed-off-by: Celeste Liu <CoelacanthusHex@gmail.com>
(cherry picked from commit 6573f0c82c09001b5f24e44df2cbe40f6f584624)

Resolves: RHEL-49810
---
 hwdb.d/70-mouse.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 20079d6350..fcd9acd2b4 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -310,6 +310,10 @@ mouse:bluetooth:v047dp8019:name:Expert Wireless TB Mouse:*
  ID_INPUT_TRACKBALL=1
  MOUSE_DPI=400@125
 
+# Kensington SlimBlade Pro trackball (via Bluetooth)
+mouse:bluetooth:v047dp80d4:name:SlimBlade Pro:*
+ ID_INPUT_TRACKBALL=1
+
 ##########################################
 # Lenovo
 ##########################################
